schema.json

```diff
@@ -0,0 +1,162 @@
+                        },
+                        "contains": {
+                          "title": "Property value must contain the specified string.",
+                          "type": "string"
+                        }
+                      }
+                    }
+                  }
+                },
+                "required": ["user"],
+                "additionalProperties": false
+              }
+            ]
+          },
+          "appId": {
+            "title": "The list of application IDs allowed for access.",
+            "type": "array",
+            "items": {
+              "type": "integer"
+            }
+          },
+          "enabled": {
+            "title": "Specifies whether the access rule is enabled or disabled.",
+            "type": "boolean"
+          },
+          "include": {
+            "title": "The list of columns to include for this access rule.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "exclude": {
+            "title": "The list of columns to exclude for this access rule.",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "require": {
+            "title": "The list of columns or property requirements for this access rule.",
+            "type": "array",
+            "items": {
+              "anyOf": [
+                {
+                  "title": "The required column for this access rule.",
+                  "type": "string"
+                },
+                {
+                  "type": "object",
+                  "additionalProperties": {
+                    "type": "object",
+                    "properties": {
+                      "equals": {
+                        "title": "Property value must be equal to the specified string.",
+                        "type": "string"
+                      },
+                      "notequals": {
+                        "title": "Property value must not be equal to the specified string.",
+                        "type": "string"
+                      },
+                      "contains": {
+                        "title": "Property value must contain the specified string.",
+                        "type": "string"
+                      }
+                    },
+                    "additionalProperties": false
+                  }
+                }
+              ]
+            }
+          },
+          "name": {
+            "title": "The name of the access rule.",
+            "type": "string"
+          },
+          "script": {
+            "title": "The custom script used for evaluating the access rule.",
+            "type": "string"
+          }
+        },
+        "required": ["type", "enabled"]
+      }
+    },
+    "entries": {
+      "title": "The entries (data) for the data source.",
+      "type": "array",
+      "items": {
+        "type": "object"
```

Comment:
- why does this say "object"?
- can you add a `data` Object attribute for each entry?

Reply:
@squallstar , I tried adding it with `data` only as we discussed. However, it is being added as a `data` column in the data source. Please take a look at the screenshot below. 
![image](https://github.com/Fliplet/fliplet-widget-data-source/assets/101858104/47ff7cdd-1f40-4b5e-b999-1edf7bad4ac1)

As per the DS post API, the `entries` object should look like below:
```
"entries": [
    {
      "full name": "John Doe",
      "email": "johndoe@example.com"
    },
    {
      "full name": "Jane Doe",
      "email": "janedoe@example.com"
    }
  ]

Reply:
@anerib apologies you're right, I wasn't expecting this to go through a single API request — as I forgot we have an endpoint to create a data source including its initial data. What's the JS API you used for this?